### PR TITLE
Favorites view general basic functionality completed

### PR DESCRIPTION
**Pull request by alext8900**
Added some mock data to be able to populate my collection view cells to make sure things worked and looked right. Implemented the collection view and collectionView methods. Was able to finally get my cells to show after I found the bug giving me problems. Fixed the layout of the cells and got the cells to look pretty decent. Will be working on adding more mock data to visualize important data for the user to view at a glance with ease at least for the mean time until we can get this API going. TODO: Need to add the functionality of a user actually being able to add a city they like to their favorites straight from the map, and also checking if a user is authenticated or not to show and hide the favorites button and add to favorites button.

### CODE REVIEW

**Review by bronsonmullens — APPROVED** [review]
Great job with the programmatic UI on the collection view. Shadows are looking good. Once we get our endpoints I'm sure we can expand on this.

File: labs-ios-starter/Profiles/Favorites/FavoritesCollectionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    @IBOutlet var cell: UICollectionViewCell!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    // MARK: UICollectionViewDataSource
+
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
```
Comment (johnnydhicks) [inline labs-ios-starter/Profiles/Favorites/FavoritesCollectionViewController.swift:28]: Don't forget to remove this

File: labs-ios-starter/Profiles/Favorites/FavoritesCollectionViewController.swift [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        return 1
+    }
+
+
+    override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of items
```
Comment (johnnydhicks) [inline labs-ios-starter/Profiles/Favorites/FavoritesCollectionViewController.swift:34]: Remove this as well

File: labs-ios-starter/Profiles/Favorites/FavoritesCollectionViewController.swift (hunk 1 above)
Comment (alext8900) [reply labs-ios-starter/Profiles/Favorites/FavoritesCollectionViewController.swift:28]: Copy that. I will get rid of these comments in the next pull. Thank you

**Review by denced — APPROVED** [review]
You figured out the merge conflict!!! Good job!

**Comment by alext8900** [comment]
No we didnt :( project is broke

**Pull request merged**